frontend/src/metabase/dashboard/components/DashboardTabs/use-dashboard-tabs.ts

```diff
@@ -28,7 +28,6 @@ export function useDashboardTabs({ location }: { location: Location }) {
 
   useSyncURLSlug({ location });
   useMount(() => dispatch(initTabs({ slug: parseSlug({ location }) })));
-  useUnmount(() => dispatch(selectTab({ tabId: null })));
```

Comment:
We already make this change in https://github.com/metabase/metabase/pull/32150, so maybe we should remove it from here and just keep the performance improvements in this PR?